### PR TITLE
New code added: js assignment 8

### PR DESCRIPTION
**Pull request by poonam-khode**

### CODE REVIEW

File: poonam/JS/Assignment8_js/que1.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>que 1</title>
```
Comment (sanketmod) [inline poonam/JS/Assignment8_js/que1.html]: change title

File: poonam/JS/Assignment8_js/que2.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (sanketmod) [inline poonam/JS/Assignment8_js/que2.html]: Change title

File: poonam/JS/Assignment8_js/que3.html [hunk 3]
```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>que 3</title>
```
Comment (sanketmod) [inline poonam/JS/Assignment8_js/que3.html]: Change title

File: poonam/JS/Assignment8_js/que1.html [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+                if (marks > 80) {
+                    alert("Enter marks less than or equals to 80.");
+                }
+                else {
+                    marks = marks + (marks / 10 * 2);
+                    arrMarks.push(marks);
+                }
+            }
+            arrMarks.forEach(myFunction);
+            function myFunction(item, index) {
```
Comment (sanketmod) [inline poonam/JS/Assignment8_js/que1.html]: Why are you creating a function if you are using forEach() method

File: poonam/JS/Assignment8_js/que3.html [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+                            arrMarks[j] = arrMarks[j + 1];
+                            arrMarks[j + 1] = temp;
+                        }
+                    }
+                }
+                document.write("<br><br>*****After Sorting (Bubble Sort)*****");
+                document.write("<br>Marks : " + arrMarks);
+            }
+            function selectionsort() {
+                var temp;
```
Comment (sanketmod) [inline poonam/JS/Assignment8_js/que3.html:55]: the logic for selection sort is wrong you are using bubble sort logic

File: poonam/JS/Assignment8_js/que4.html [hunk 6]
```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Merge Sort Method 1</title>
+        <script>
```
Comment (sanketmod) [inline poonam/JS/Assignment8_js/que4.html:8]: don't write entire logic in head section

File: poonam/JS/Assignment8_js/que4.html [hunk 7]
```diff
@@ -0,0 +1,79 @@
...
+            }
+            for (; j <= 4;) {
+                result[k] = sectionB[j];
+                k++;
+                j++;
+            }
+            document.write("<br>Result array (after sorting) : " + result);
+        </script>
+    </head>
+    <body>
```
Comment (sanketmod) [inline poonam/JS/Assignment8_js/que4.html]: Format document properly

**Review by sanketmod — CHANGES_REQUESTED** [review]

**Review by MKadeer17 — APPROVED** [review]

**Review by sanketmod — APPROVED** [review]

File: poonam/JS/Assignment8_js/que3.html [hunk 8]
```diff
@@ -0,0 +1,125 @@
...
+                else {
+                    marks = marks + (marks / 10 * 2);
+                    arrMarks.push(marks);
+                }
+                document.write("<br> Marks of subject " + (i + 1) + ": " + arrMarks[i]);
+            }
+            let select = parseInt(prompt("1 – Bubble sort (Press 1)\n2 – Insertion sort (Press 2)\n3 – selection sort(Press 3)\n4 – Quick Sort(Press 5)\n5 – Exit (Press X or Cancel or 5)"));
+            switch (select) {
+                case 1:
+                    bubblesort();
```
Comment (Snehal-113) [inline poonam/JS/Assignment8_js/que3.html:26]: the array needs to be passed as parameter to all the sorting algo functions.
bubblesort(arrMarks);

File: poonam/JS/Assignment8_js/que3.html [hunk 9]
```diff
@@ -0,0 +1,125 @@
...
+                case 3:
+                    insertionsort();
+                    break;
+                case 4:
+                    quicksort();
+                    break;
+                default:
+                    break;
+            }
+            function bubblesort() {
```
Comment (Snehal-113) [inline poonam/JS/Assignment8_js/que3.html:40]: bubblesort(arrMarks){

**Review by Snehal-113 — APPROVED** [review]

**Pull request merged**